### DISCUSSION TITLE
Selecting notes with more than one tag

### DISCUSSION THREAD

**Discussion by grvsd** [standardnotes/forum#2515] [Suggestions]
Wouldn't it be nice if one could press Ctrl and pick several tags in the left sidebar (of the desktop app) to select notes with corresponding tags? What do you think?
As to the Smart Tags, this feature seems to be too powerful for such a task, I think. Though still useful for saved searches.

**Comment by atmoio** [comment]
Yeah that would be cool :) But tricky for now. But I think the smart tag architecture will make this feature, if it ever comes (probably in Folders), much easier. Perhaps Folders can do this by creating a temporary smart tag dynamically once a user selects more than one tag. That would be an interesting solution.

**Comment by kvn80** [comment]
Oh Yes, that would be very appreciated.
Maybe you could start by permitting to search by tag into the research bar (with @ or #) in order to put multiple tags on the same research.
It seems necessary for me to have this option for working in a real Tag mode instead of folder and subfolder way.

**Comment by atmoio** [comment]
That's also an interesting idea. Will keep that in mind, thanks!

**Comment by kvn80** [comment]
I found a efficient temporary solution :
I copie (without Quick Tags activated) or rewrite the tag list : "#work #car #repair" , on the first line of the note.
And so I can type something like "#rep #wo #ca" and find rapidely THE note i'm looking for.
To simplify that solution, you just have to include all the tags in the search engine.
I think that should not a be a big job, but i'm not developer ...
Thanks for making my new favorite note app

**Comment by atmoio** [comment]
Ah, nice solution! Hadn't considered that. Yeah, would be good if the search took tags into consideration. Shouldn't be too difficult, but might decrease speed. Will have to play around with it.

**Comment by kvn80** [comment]
Hi @mobitar 
As it seems not to be a hard job to include the tags in the search engine, do you think it is possible to add this possibility ?
I'm always duplicating the tags in the notes ... when I remember to do that correctly.

**Comment by atmoio** [comment]
Definitely not always trivial to implement stuff like this. Nothing is ever easy. I can say for sure this wouldn't be just a couple lines of code, but a huge change to search. Still need to consider which direction we're going with search: keep it simple, but create search extensions, or go all out on the core search.

**Comment by mmlmrx** [comment]
I would like to second the vote for including tags in search.  Searching for tags using # is a pretty common design concept now, which means those could live in their own separate search index - not included with the existing notes content index.

**Comment by atmoio** [comment]
@mmlmrx can you describe how you would use this behavior if it existed?

**Comment by mmlmrx** [comment]
Well, an example would be games I have played.  I would have a main tag of #Game, then for each game I play, I would add a note with tags of #platform, #year, and #Completed.  So, if I want to see what games I played in 2018, I would search using #game #2018.  

From there, it could be expanded to an Advanced Search by combining the tag index and content index - so if they search for both tags and some untagged text, query the tag index first to limit the scope of the second (content) search.

**Comment by johnmcn1** [comment]
One of the first things I tried to do when I starting using tags. I hope this request has not been forgotten.

**Comment by JaspalSuri** [comment]
Hi @johnmcn1, thanks for adding your +1 to this request. While this request hasn't been forgotten, implementing it currently isn't a part of our near-term plans.

**Comment by myreli** [comment]
Relates to https://github.com/standardnotes/forum/discussions/2566
